Lib/multiprocessing/spawn.py

```diff
@@ -100,25 +100,22 @@ def spawn_main(pipe_handle, parent_pid=None, tracker_fd=None):
 
         if parent_pid is not None:
             source_process = _winapi.OpenProcess(
-                _winapi.PROCESS_DUP_HANDLE, False, parent_pid)
+                _winapi.PROCESS_ALL_ACCESS, False, parent_pid)
```

Comment:
Nice!

Reply:
Out of curiosity, is it possible to narrow the permissions? (for example `PROCESS_ALL_ACCESS + SYNCHRONIZE`?)

Reply:
You mean `PROCESS_DUP_HANDLE` and `SYNCHRONIZE`? I tried a few days ago and I got an `access denied`.

Reply:
Ok. Perhaps @zooba has more insight about this.

Reply:
Update: after a good night sleep, `PROCESS_DUP_HANDLE` | `SYNCHRONIZE` do work.